terrasync/vendor/modules.txt

```diff
@@ -0,0 +1,2 @@
+# github.com/pkg/errors v0.8.1
+github.com/pkg/errors
```

Comment:
Any reason for this?

Reply:
The file was generated by `go mod vendor` not super sure how it's used though